mindsdb/api/http/namespaces/entitites/predictor_status.py

```diff
@@ -10,6 +10,7 @@
     # other attributes
     'is_active': fields.Boolean(required=False, description='Only one predictor by public_name can be active'),
     'data_source': fields.String(required=False, description='The data source it\'s learning from'),
+    'data_source_name': fields.String(required=False, description='The name of the datasource it\'s learning from'),
```

Comment:
is this not the same as 'data_source'?

Reply:
No, in general `data_source` points to the .csv file with data